static/vendor/reveal/js/config.js

```diff
@@ -18,10 +18,6 @@ export default {
 	// Display presentation control arrows
 	controls: true,
 
-	// Help the user learn the controls by providing hints, for example by
```

Comment:
I don't think this was an intended change? This code belongs to the slides library

Reply:
You are right, thanks for pointing it out!